### ISSUE TITLE
Question: change get functions

### ISSUE THREAD

**Issue by elenril2** [openviess/PyViCare#234]
Hello,

I want to change get consumpionts functions in GazBoiler class to match my device Vitodens 100-W. I must change ```self.service.getProperty("heating.gas.consumption.heating")["properties"]["unit"]["value"]``` to ```self.service.getProperty("heating.gas.consumption.summary.heating")["properties"]["unit"]["value"]```.
But when I do this directly in PyViCareGazBoiler.py I've got errors:
```
Traceback (most recent call last):
  File "vicare.py", line 56, in <module>
    print(t.getGasConsumptionHeatingUnit())
  File "/usr/local/lib/python3.7/dist-packages/PyViCare-2.13.0.post2+git.c95094c3-py3.7.egg/PyViCare/PyViCareUtils.py", line 45, in feature_flag_wrapper
  File "/usr/local/lib/python3.7/dist-packages/PyViCare-2.13.0.post2+git.c95094c3-py3.7.egg/PyViCare/PyViCareUtils.py", line 38, in wrapper
  File "/usr/local/lib/python3.7/dist-packages/PyViCare-2.13.0.post2+git.c95094c3-py3.7.egg/PyViCare/PyViCareGazBoiler.py", line 22, in getGasConsumptionHeatingUnit
    return self.service.getProperty("heating.gas.consumption.summary.heating")["properties"]["unit"]["value"]
  File "/usr/local/lib/python3.7/dist-packages/PyViCare-2.13.0.post2+git.c95094c3-py3.7.egg/PyViCare/PyViCareCachedService.py", line 26, in getProperty
    return readFeature(entities, property_name)
  File "/usr/local/lib/python3.7/dist-packages/PyViCare-2.13.0.post2+git.c95094c3-py3.7.egg/PyViCare/PyViCareService.py", line 17, in readFeature
    if(feature is None):
PyViCare.PyViCareUtils.PyViCareNotSupportedFeatureError: heating.gas.consumption.heating

```

What I'm doing wrong, is there a cache that sends old property_name heating.gas.consumption.heating instead of heating.gas.consumption.summary.heating?

**Comment by woehrl01** [comment]
Hi @elenril2,

There isn't an active cache. It's only stored in memory so after a restart it's gone.

As I haven't seen your new property on the wild. Do you have a dump to share?

**Comment by elenril2** [comment]
Hello @woehrl01, thanks for answer. Here is my dump: https://github.com/somm15/PyViCare/pull/225

**Comment by damianbro** [comment]
Hello @elenril2  @woehrl01 
I made some changes according to your dump. I made new json dump today and there are some additional data.
I made also new methods to get consumption with new API properties. Here is a Pull Request for that:
[Pull Request #235 ](https://github.com/somm15/PyViCare/pull/235)

**Comment by woehrl01** [comment]
Thank you for your contribution. I just published 2.15.0
